tracer/test/Datadog.Trace.Tests/DataStreamsMonitoring/DataStreamsManagerTests.cs

```diff
@@ -138,7 +138,7 @@ public void WhenEnabled_SetCheckpoint_SetsSpanTags()
         var dsm = GetDataStreamManager(true, out _);
         var span = new Span(new SpanContext(traceId: 123, spanId: 456), DateTimeOffset.UtcNow);
 
-        span.SetDataStreamsCheckpoint(dsm,  CheckpointKind.Produce, new[] { "direction:out" });
+        span.SetDataStreamsCheckpoint(dsm,  CheckpointKind.Produce, new[] { "direction:out" }, 100, 0);
```

Comment:
I'm not sure how easy its to do, but AFAICT we don't have any tests that are checking 
- the payload bytes are aggregated correctly
- the edge time is calculated correctly

You've fixed the existing tests so they pass (great!) but I don't think you've added any new tests or assertions? It's possible I've just not followed correctly, but otherwise I think we should add some to test the new/changed functionality 🙂